airbyte-integrations/connectors/source-hubspot/source_hubspot/schemas/contacts_list_memberships.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "canonical-vid": {
+      "type": ["null", "integer"]
```

Comment:
Could it really be `null` (or absent)?

Reply:
@sergei-solonitcyn there is not enough information about it in documentation. In another streams null value is supposed.